host.go

```diff
@@ -206,9 +206,27 @@ func (h *Host) ConfigureAuth() error {
 		return nil
 	}
 
-	ip, err := h.Driver.GetIP()
-	if err != nil {
-		return err
+	var (
+		ip         = ""
+		ipErr      error
+		maxRetries = 4
+	)
+
+	for i := 0; i < maxRetries; i++ {
+		ip, ipErr = h.Driver.GetIP()
```

Comment:
Shouldn't the return of the error show it from create?  I can debug it here
too if needed.

On Tuesday, February 24, 2015, Nathan LeClaire <notifications@github.com>
wrote:

> In host.go
> <https://github.com/docker/machine/pull/602#discussion_r25288048>:
>
> > @@ -206,9 +206,27 @@ func (h *Host) ConfigureAuth() error {
> >  		return nil
> >  	}
> >
> > -	ip, err := h.Driver.GetIP()
> > -	if err != nil {
> > -		return err
> > +	var (
> > +		ip         = ""
> > +		ipErr      error
> > +		maxRetries = 4
> > +	)
> > +
> > +	for i := 0; i < maxRetries; i++ {
> > +		ip, ipErr = h.Driver.GetIP()
>
> Consider log.Debugfing this ipErr since if something funky happens we
> want to know about it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/machine/pull/602/files#r25288048>.
>